pkg/handlers/app.go

```diff
@@ -55,6 +57,13 @@ type SendCustomAppMetricsRequest struct {
 
 type CustomAppMetricsData map[string]interface{}
 
+type SendAppInstanceTagsRequest struct {
+	Data struct {
+		IsForced bool                   `json:"isForced"`
+		Tags     map[string]interface{} `json:"tags"`
```

Comment:
I still don't think this should allow interfaces. Tag values will be stored as strings, and we will not be able to restore the original data type.

Reply:
Fixed.  Used `map[string]string` and introspected the Unmarshal error.

```
╰─$ curl -X POST localhost:3000/api/v1/app/instance-tags -d '{"data": {"isForced": true, "tags": {"name": 1}}}'
number not supported, only string values are allowed on instance-tags
```